src/combine.cpp

```diff
@@ -500,11 +500,11 @@ static chunk_t *get_d_template_types(ChunkStack &cs, chunk_t *open_paren);
 static bool chunkstack_match(ChunkStack &cs, chunk_t *pc);
 
 
-enum PLBfound
+enum class plb_found_e : uint8_t
```

Comment:
It is really a bad name.
I found do_pl declared at line 5092 as int, with some "enum" values.
P stand for Protocol.
May be you could propose a better name. It would be fine.